src/Algebra/Properties/CommutativeSemigroup.agda

```diff
@@ -0,0 +1,121 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some theory for commutative semigroup
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra using (Semigroup)
+open import Algebra.FunctionProperties using (Commutative)
+
+module Algebra.Properties.CommutativeSemigroup {ℓ ℓ=} (H : Semigroup ℓ ℓ=)
```

Comment:
Please lay out the module parameters as per the style guide.

Reply:
Done.